tests/memory/enc/fragment.c

```diff
@@ -80,7 +80,7 @@ static int _malloc_free_random_size(int times, unsigned int seed)
     srand(seed);
     for (i = 0; i < times; i++)
     {
-        size = _randx(_get_heap_size());
+        size = _randx(_get_heap_size() - 23);
```

Comment:
Thanks for investigate this! It seems that the problem is that the test hits a corner case (i.e., allocating certain large size heap). Rather than avoiding hitting the case, do you think we can fundamentally address the problem? I'd suggest as follows.
- Investigate if there is a fix (maybe certain configuration to the dlmalloc).
- Add tests to explicitly test this corner case. In this case, we should know what would be the expected behavior.

Reply:
@larrywxyz  Can you address this comment?